homeassistant/components/automation/__init__.py

```diff
@@ -290,6 +305,14 @@ async def reload_service_handler(service_call: ServiceCall) -> None:
         schema=vol.Schema({}),
     )
 
+    async_register_admin_service(
+        hass,
+        DOMAIN,
+        SERVICE_RELOAD_AUTOMATION,
+        reload_single_service_handler,
+        schema=vol.Schema({CONF_ID: str}),
```

Comment:
This should be entity IDs maybe?

Reply:
No it shouldn't, because the entity_id is not stable.
We could perhaps allow passing in either the automation ID (for use by frontend) or an entity_id (for manual use).

Reply:
Entity ID doesn’t need to be stable. It just needs to be there when it’s called. The nice thing about entity ID is that we can use an entity selector in the Frontend.

Reply:
I also think that we should do entity ID because it’s how most other services work, and it’s nice to be consistent. Otherwise users have to learn a new concept.